qa/L0_server_status/server_status_test.py

```diff
@@ -217,6 +219,17 @@ def test_model_specific_infer(self):
             except InferenceServerException as ex:
                 self.assertTrue(False, "unexpected error {}".format(ex))
 
+    def test_json_server_status(self):
```

Comment:
The test case content doesn't make sense. You are still receiving protobuf message from server instead of JSON. Here you just testing the correctness of protobuf's JSON convertor...

Reply:
Because of the way our client library is written, it defaults to using binary as the format. The same converter is what we are using inside the server to return the server status as a JSON.

Reply:
So what is the point of having this test case? It's not hitting the code change in this PR, right?

Reply:
Fixed test coverage in following commit